servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/RoundRobinLoadBalancer.java

```diff
@@ -607,6 +606,7 @@ private Completable doClose(final Function<? super C, Completable> closeFunction
 
         @Override
         public String toString() {
+            final ConnState connState = this.connState;
```

Comment:
Local variable helps to make sure, `connState.state` and `connState.connections` relate to the same `connState` instance.